app/feats/insurance_claim_feat.py

```
@@ -1,175 +1,561 @@
+        claim_subject=claim_subject,
         correlation_id=correlation_id,
+        idempotency_key=idempotency_key,
     )
-    return InsuranceClaimResult(
-        claim_id=claim.claim_id,
-        status=claim.status.value,
-        message=f"{claim_type.replace('_', ' ').title()} claim submitted.",
+
+
+@feat_shell("FEAT-STOR-003")
+def _submit_insurance_claim_impl(
```

Ruff (0.16.0)

[warning] Too many return statements (13 > 6)

(PLR0911)